app/templates/checkout/complete.html

```diff
@@ -183,7 +183,7 @@ <h4 class="form-title">
               </form>
             </div>
             <div class="pull-right">
-              <button type="button" go-to-payment ng-disabled="customerForm.$invalid || addressForm.$invalid" class="btn btn-lg btn-success">Prosseguir para pagamento</button>
+              <button type="button" go-to-payment ng-disabled="customerForm.$invalid || addressForm.$invalid" class="btn btn-lg btn-success" disable-on-load="complete">Prosseguir para pagamento</button>
```

Comment:
Já existia e é usada em vários outros lugares. Quando chamamos uma função que carrega algo, usamos `this.loader.start('complete');` que faz aparecer o `spinner` no meio da tela. 

Só que, além disso (novidade pra mim), o `loader.start` que o @diogobeda fez também pode ser inteligentemente usado pra outras coisas ocorrerem enquanto se está carregando algo. Nesse aso, desabilitamos o botão pra impedir outros cliques.